test/parallel/test-async-wrap-getasyncid.js

```diff
@@ -212,9 +213,10 @@ if (common.hasCrypto) {
   const TCP = process.binding('tcp_wrap').TCP;
   const tcp = new TCP();
 
-  const ca = fs.readFileSync(common.fixturesDir + '/test_ca.pem', 'ascii');
-  const cert = fs.readFileSync(common.fixturesDir + '/test_cert.pem', 'ascii');
-  const key = fs.readFileSync(common.fixturesDir + '/test_key.pem', 'ascii');
+  const ca = fs.readFileSync(fixturesPath('test_ca.pem'), 'ascii');
+  const cert = fs.readFileSync(fixturesPath('test_cert.pem'), 'ascii');
+  const key = fs.readFileSync(fixturesPath('test_key.pem'), 'ascii');
```

Comment:
no... see above. I left these because the keys being read require the additional encoding arg and but the paths do not include the `keys` directory.